web/manifest/manifest.go

```diff
@@ -50,7 +50,7 @@ func ReadyForAppCreation(envFile string, chkFns ...func() error) error {
 }
 
 func CreateNewQuickFeedApp(srvFn web.ServerType, httpAddr, envFile string) error {
```

Comment:
Should this line in `StartAppCreationFlow` below:
```go
log.Printf("Go to https://%s/manifest to install the QuickFeed GitHub App.\n", env.Domain())
```
also use `DomainWithPort()`?

Reply:
Also, this function has a test in `web/manifest/manifest_test.go` that calls it like this:
```go
if err := manifest.CreateNewQuickFeedApp(web.NewDevelopmentServer, ":443", envFile); err != nil {
```
Should this test be adjusted?